var/spack/repos/builtin/packages/sol2/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class Sol2(CMakePackage):
+    """a C++ <-> Lua API wrapper with advanced features and top notch performance"""
+
+    homepage = "https://github.com/ThePhD/sol2"
+    url = "https://github.com/ThePhD/sol2/archive/v3.0.3.tar.gz"
```

Comment:
There is already a `lua-sol2` package.

FWIW in my experience the easiest way to know if a Spack package has been made yet is to look for the "identifying" part of the URL (In this case I recursively grep'ed through our builtin package repo for "sol2")

Reply:
I looked with spack list, and I didn't see it.  I could have been added after I first packaged LibPressio-Opt/OptZConfig since that was one of the earlier packages I added.

Reply:
I've now switched to the upstream lua-sol2